tests/cc56/input.dat

```diff
@@ -0,0 +1,58 @@
+#! EOM-CCSD/6-31g excited state transition data for water cation
+
+molecule h2o {
+  1 2 
+  O
+  H 1 1 
+  H 1 1 2 104.5
+}
+
+set {
+  reference uhf 
+  basis cc-pVDZ
+  freeze_core true
+  roots_per_irrep [0, 1, 1, 0]
+}
+
+wfn = properties('eom-ccsd', properties=['oscillator_strength'], return_wfn=True)[1]
+
+### ccdensity checks
+
+refnre     =   8.8014655645673745 # TEST
+refscf     = -75.6332569214465593 # TEST
+refccsd    = -75.803716136508982  # TEST
+refoverlap =   0.95259415503      # TEST
+
+compare_values(refnre, h2o.nuclear_repulsion_energy(), 6, "Nuclear Repulsion Energy") # TEST
+compare_values(refscf, wfn.variable("SCF TOTAL ENERGY"), 6, "SCF Energy") # TEST
+compare_values(refccsd, wfn.variable("CCSD TOTAL ENERGY"), 6, "CCSD Energy") # TEST
+compare_values(refoverlap, wfn.variable("LEFT-RIGHT CCSD EIGENVECTOR OVERLAP"), 5, "Left-Right CCSD Overlap") # TEST
+
+### Excitation energy checks
+
+ref1B1 = -75.325751117648 # TEST
+ref0A2 = -75.260476163556 # TEST
+
+compare_values(ref1B1, core.variable("CCSD ROOT 1 (B1) TOTAL ENERGY"), 6, "Root 1 (A1) Energy") # TEST
+compare_values(ref0A2, core.variable("CCSD ROOT 0 (A2) TOTAL ENERGY"), 6, "Root 0 (A2) Energy") # TEST
+
+### Transition checks
+
+test_data = { # TEST
+"ROOT 0 (B1) -> ROOT 0 (A2)": {"OSCILLATOR STRENGTH (LEN)": 0.00407528, "EINSTEIN A (LEN)": 3.86413231e+07, "EINSTEIN B (LEN)": 1.36907729e+18}, # TEST
```

Comment:
I hate to suggest it now, but what are your thoughts on `ROOT n (h)` always meaning all-root ordering and `ROOT n OF (h)` always meaning within-symmetry-ordering?

Reply:
I hate to admit it now, but that _is_ cleaner.

We're keeping "ROOT n OF (h)", but do you want "ROOT n (h)" notation, to boot?

Reply:
"OF" or "IN", do you think?

> do you want "ROOT n (h)" notation, to boot?

I don't know. may as well for now. up to you.

Reply:
You're right, "IN" is better.

I think the plan _for right now_ is to keep with the current notation, then have a cleanup PR once the series is over (we're very close), where switch all excited state variables to "IN" notation and add the new "ROOT n (h)" notation as well.